### PR TITLE
[Refactor] Lifecycle Page

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: Yash Kamboj <yashkamboj29@gmail.com>

**Description**
Refactored the Lifecycle Page without the data 
Screenshot (183)

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryMetrics/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+// import {Paper, Typography, Button } from "@material-ui/core";
```
Comment (adithyaakrishna) [inline ui/components/MesheryMetrics/index.js:2]: I guess if this import is not needed, we can remove it 👀

File: ui/components/ReactSelectWrapper.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CreatableSelect from 'react-select/creatable';
+import { styled } from "@mui/material/styles";
+import { MenuItem,TextField,Typography, Chip} from "@mui/material";
+
+
+const CustomCreatableSelect = styled(CreatableSelect)(({ theme }) => ({
+// marginLeft: "4rem",
+// width: "70%",
```
Comment (adithyaakrishna) [inline ui/components/ReactSelectWrapper.js:9]: Same applies for this commented lines as well

**Review by adithyaakrishna — APPROVED** [review]
LGTM

**Comment by adithyaakrishna** [comment]
@YashKamboj Let's wait for @sudo-NithishKarthik or @Chadha93 's review on this and then will Merge it 💯

**Comment by Chadha93** [comment]
> @YashKamboj Let's wait for @sudo-NithishKarthik or @Chadha93 's review on this and then will Merge it 100

@adithyaakrishna Let's hold this and other refactoring PRs until we get back with the qualifying criteria for merging refactoring based Prs.

**Comment by YashKamboj** [comment]
@Chadha93 @sudo-NithishKarthik Merging this pr would be really helpful , as this pr also have the `SelectWrapperComponent` which is used in many other components (including the ones i have opened pr for, don't worry for the merge conflicts that might cause due to `SelectWrapper` as i haven't added it in other prs)

File: ui/components/MesheryMetrics/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import { PaperWithoutTitle } from "@/components/Paper";
+import { MetricsButton } from "@/components/Button"
+import AddIcon from '@mui/icons-material/Add';
+import {Typography} from "@mui/material";
+
+function MesheryMetrics(){
+
+    const CustomPaperWithoutTitle = styled(PaperWithoutTitle)(() => ({
```
Comment (sudo-NithishKarthik) [inline ui/components/MesheryMetrics/index.js:10]: this `CustomPaperWithoutTitle` does not mean anything 😅 . For these kind of use cases, please use `sx` prop.

Comment (sudo-NithishKarthik) [reply ui/components/MesheryMetrics/index.js:10]: Not that there's anything wrong with it, it's just that it makes it easy to understand.

File: ui/components/ReactSelectWrapper.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+// import Select from 'react-select';
+import CreatableSelect from 'react-select/Creatable';
```
Comment (sudo-NithishKarthik) [inline ui/components/ReactSelectWrapper.js:3]: Let us use `mui`, not `react-select`. `mui` can do all this now.

File: ui/pages/management.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
+import React, {useState} from "react";
+import { Grid,TextField, MenuItem,  Chip} from "@mui/material";
+import { PaperWithoutTitle, PaperWithTitle } from "@/components/Paper";
+import { styled } from "@mui/material/styles";
+const {img} = "../../docs/assets/img/adapters/linkerd/linkerd-adapter.png" 
+import MesheryMetrics from "@/components/MesheryMetrics" 
+import ReactSelectWrapper from "@/components/ReactSelectWrapper"
+
+export default function Lifestyle() {
```
Comment (sudo-NithishKarthik) [inline ui/pages/management.js:9]: `Lifecycle`

File: ui/pages/management.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+ 
+  const CustomChip = styled(Chip)(({ theme }) => ({
```
Comment (sudo-NithishKarthik) [inline ui/pages/management.js:11]: lets use `sx` prop for this

**Review by sudo-NithishKarthik — COMMENTED** [review]
@YashKamboj , please address these comments in your next pr.

File: ui/pages/management.js (hunk 5 above)
Comment (YashKamboj) [reply ui/pages/management.js:9]: 😅 github co-piltot

**Pull request merged**